pasture-algorithms/Cargo.toml

```diff
@@ -13,4 +13,8 @@ readme = "README.md"
 
 [dependencies]
 pasture-core = { version = "=0.1.0", path = "../pasture-core" }
-anyhow = "1.0.34"
\ No newline at end of file
+anyhow = "1.0.34"
+pasture-derive = { version = "=0.1.0", path = "../pasture-derive" }
+rand = "0.8.3"
+pasture-io = { version = "=0.1.0", path = "../pasture-io" }
```

Comment:
Please don't add a mandatory reference to pasture-io in the algorithms crate. Pasture-io is quite heavyweight because of the other libraries it includes, and it should be possible to run all algorithms purely on `PointBuffer` types, without any need for using I/O. If it is really mandatory, the pasture-io include AND the algorithm which requires it should be behind a feature flag